docs/src/integrations/activity/reference.md

```diff
@@ -2,364 +2,715 @@
+| `environment.variable.delete` | A variable has been deleted. |
+| `environment.variable.update` | A variable has been modified. |
+| `environment.update.http_access` | HTTP access rules for an environment have been modified. |
+| `environment.update.smtp` | Sending of emails has been enabled/disabled for an environment. |
+| `environment.update.restrict_robots` | The block-all-robots feature has been enabled/disabled. |
+| `environment.subscription.update` | The production environment has been resized because the subscription has changed. There's no content changes. |
+| `environment.cron` | A cron task just completed. |
+| `environment.source-operation` | A source operation triggered and has completed. |
+| `environment.certificate.renewal` | An environment's SSL certificate has been renewed. |
+
+Note that changes regarding user access to an environment/project aren't available anymore through the environment object.
+
+For an overview of the different actions that can take place, see [the type](#type).
+
+#### `Integration` event value
+
+This value describes the action that happened on a project's integration, its value can be any from the following table.
+
+| Name | Description |
+|------|-------------|
+| `integration.bitbucket.fetch` | Changes in BitBucket repository have been pulled. |
+| `integration.bitbucket.register_hooks` | Integration hook have been registered on BitBucket. |
+| `integration.bitbucket_server.fetch` | Changes in BitBucket repository have been pulled. |
+| `integration.bitbucket_server.register_hooks` | Integration hook have been registered on BitBucket. |
+| `integration.github.fetch` | Changes in GitHub repository have been pulled. |
+| `integration.gitlab.fetch` | Changes in GitLab repository have been pulled. |
+| `integration.health.email` | Health event sent by email. |
+| `integration.health.pagerduty` | Health event sent to PagerDuty. |
+| `integration.health.slack` | Health event sent to slack. |
+| `integration.webhook` | Webhook triggered. |
+| `integration.script` | An activity script has run. |
+
+For an overview of the different actions that can take place, see [the type](#type).
 
 ### `environments`
 
-An array listing the environments that were involved in the activity.
-This is usually single-value.
+An array listing the `environments` that were involved in the activity.
+This is usually a single-value element.
+
+### `state`
+
+The current state of the activity.
+Its value can be `pending`, `in_progress`, or `complete`.
+
+### `completion_percent`
+
+The completion percentage value of the activity.
 
 ### `result`
 
-Whether the activity was completed successfully or not.
-It should be `success` if all went as planned.
+Whether the activity itself completed successfully or not.
+Its value is `success` if all went as planned.
+The result doesn't take into account if your cron task executed properly or if the (re)deploy ran successfully.
```

Comment:
I'm just not certain where the line between the activity and anything else is. What's the activity in that case? Would it just be because someone didn't include `-e` in the command or something else? I know deploy commands can fail and still be marked as success. But not sure that means "all went as planned"